docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -2,30 +2,89 @@
 
 ## Introduction
 
-Dasharo Tools Suite (DTS) is a set of tools for deploying, updating and
-maintaining firmware on Dasharo supported devices. DTS is integrated into the
-firmware in Dasharo supported devices and can be used to update the firmware
-even when no OS is installed.
+Dasharo Tools Suite (DTS) is a set of tools running in a minimal Linux
+environment, with a goal of deploying, updating and maintaining firmware on
+Dasharo supported devices. For example, it can be used to update the firmware
+on a device, even when no OS is currently installed.
 
-## Requirements
+## Distribution methods
 
-- Dasharo device with DTS functionality integrated
-- Wired network connection
-- Secure Boot disabled
+It can be distributed in various ways. Currently, there are two distribution
+options:
 
-## Launching DTS
+* bootable USB stick image.
+* bootable over network (iPXE),
+
+## Flavors
+
+There is a common base, but there might be multiple flavors of the DTS images.
+Currently, there are:
+
+* CE
+    - `Dasharo HCL Report` - generates a package with logs containing hardware
+      information
+    - flashrom, fwupd, and many more useful tools
+    - can drop to shell to update the firmware manually
+* OEM
+    - on top of that, provides tools for automatic firmware deployment and
+      rollback (switching to Dasharo back and forth)
+
+## Bootable USB stick
+
+This documentation is compatible with the `v0.3.0` version of the DTS.
+
+### Requirements
+
+* USB stick (at least 1GB)
+* Wired network connection
+* Secure Boot disabled
+* [DTS CE v0.3.0 downloaded](https://cloud.3mdeb.com/index.php/s/Q7pAppm4gnRCef9/download)
+
+### Launching DTS
+
+* Flash the downloaded `dts-base-image-ce-v0.3.0.wic.gz` image onto USB stick.
+  - you can use cross-platform GUI installer - [Etcher](https://www.balena.io/etcher/)
+  - you can also use `dd` to flash from command line
+
+```bash
+gzip -cdk dts-base-image-ce-v0.3.0.wic.gz | sudo dd of=/dev/sdX bs=16M status=progress
```

Comment:
Using bs of 16MB will certainly cause some post-dd sync of the data on the stick. If someone pulls out the stick right after dd, data corruption is guaranteed. The typical MTU for USB is 512 bytes. I would advise adding a next step which will instruct to safely eject the drive from the system using the desktop manager and wait until the remaining writes actually occur.

Reply:
`conv=fdatasync` should handle this for us:

```suggestion
gzip -cdk dts-base-image-ce-v0.3.0.wic.gz | sudo dd of=/dev/sdX bs=16M status=progress conv=fdatasync
```